pydantic_evals/pydantic_evals/evaluators/llm_as_a_judge.py

```diff
@@ -83,6 +83,8 @@ def _stringify(value: Any) -> str:
     if isinstance(value, str):
         return value
     try:
+        # If the value can be serialized to JSON, use that.
+        # If that behavior is undesirable, the user could manually call repr on the arguments to the judge_* functions
         return to_json(value).decode()
```

Comment:
I at first thought that we should remove the JSON serialization here too, and I'm still not confident we shouldn't, but  you do have the option to call repr on the inputs before passing them to the judge_ functions. Admittedly, you'd need to make a custom evaluator to do that, but 🤷‍♂️ I'm not convinced this is wrong yet. Would be happy to change it to just use repr rather than json serializing if others feel strongly that that's better though.